tests/runtime/engine/parser.py

```diff
@@ -123,20 +123,33 @@ def __read_test_struct(test, test_suite):
         will_fail = False
         new_pidns = False
         skip_if_env_has = None
+        prev_item_name = ''
 
         for item in test:
+            if item[:len(prev_item_name) + 1].isspace():
+                # Whitespace at beginning of line means it continues from the
+                # previous line
+                line = item[len(prev_item_name) + 1:]
+                if prev_item_name == 'PROG':
+                    prog += line
```

Comment:
Should also add newline?
```suggestion
                    prog += line + "\n"
```